---
layout: page
title: Program BioVis@Vis
permalink: /program_ieee/
back_title: IEEE VIS 2016
back_url: ieeevis
---

### Keynote Speaker
<div class="talk">
    <div class="ttitle">Visualization: The power of alternate representations
        <!--{% if talk.slides %}
        <span class="tspeaker" style="float: right;"><a href="{{ site.baseurl }}/files/{{talk.slides}}">Download
            Slides</a></span>
        {% endif %}-->
    </div>
    <div><span class="tspeaker"><a href="http://pages.cpsc.ucalgary.ca/~sheelagh/wiki/pmwiki.php">Sheelagh Carpendale</a></span>, <span>University of Calgary</span></div>

    <div class="tportrait"><img src="{{ site.baseurl}}/images/speakers/sCarpendale.jpg" style="width: 250px;" alt="S.Carpendale">
    </div>

    <div class="tbioabstract">

        <div class="tabstract"><b>Abstract:</b>
        To visualize data one of the first steps is to develop a visual representation. This representation is a result of a mapping by which the data can be specified. Much has been said of about the power of these visual representations. Simon (1981) said that solving a problem is simply a matter of representing so as to make the solution transparent – implying that finding the right representation solves the problem. Card et al. (1998) said that interactive visual representations can amplify our cognition – can in effect make us smarter.  In spite of this, the small box in the visualization creation pipeline that signifies the development of the visual representation remains one of the least unpacked. Through examples from my own work and others’, I will discuss the power and potential of alternate visual representations.
        </div>

        <div class="tbio"><b>Bio:</b>
        Sheelagh Carpendale is a Professor in Computer Science at the University of Calgary where she holds a Canada Research Chair in Information Visualization and NSERC/AITF/SMART Technologies Industrial Research Chair in Interactive Technologies. She has many received awards including the E.W.R. NSERC STEACIE Memorial Fellowship; a BAFTA (British Academy of Film & Television Arts Interactive Awards); an ASTech Innovations in Technology award; and the CHCCS Achievement Award, which is presented periodically to a Canadian researcher who has made a substantial contribution to the fields of computer graphics, visualization, or human-computer interaction. She has served in such roles as Papers, Program, or Conference Chair for IEEE InfoVis, ACM Interaction Tabletops and Surfaces, and Computational Aesthetics and has received both the IEEE and ACM recognition of service awards. She is a co-director of the Interactions Lab and leads the Innovations in Visualization (InnoVis) research group and initiated the interdisciplinary graduate program, Computational Media Design. Her research on information visualization, large interactive displays, and new media draws on her dual background in Computer Science and Visual Arts.</div>
    </div>
</div>



### Primer Speaker
<div class="talk">
    <div class="ttitle">Statistical dances: why no statistical analysis is reliable and what to do about it
        <!--{% if talk.slides %}
        <span class="tspeaker" style="float: right;"><a href="{{ site.baseurl }}/files/{{talk.slides}}">Download
            Slides</a></span>
        {% endif %}-->
    </div>
    <div><span class="tspeaker"><a href=" http://www.aviz.fr/badstats">Pierre Dragicevic</a></span>, <span> Inria, France </span></div>

    <div class="tportrait"><img src="{{ site.baseurl}}/images/speakers/pDragicevic.jpeg" style="width: 250px;" alt="P.Dragicevic">
    </div>

    <div class="tbioabstract">

        <div class="tabstract"><b>Abstract:</b>
        It is now widely recognized that we need to improve the way we report empirical data in our scientific papers. More formal training in statistics is not enough. We also need good "intuition pumps" to develop our statistical thinking skills. In this talk I explore the basic concept of statistical dance. The dance analogy has been used by Geoff Cumming to describe the variability of p-values and confidence intervals across replications. I explain why any statistical analysis and any statistical chart dances across replications. I discuss why most attempts at stabilizing statistical dances (e.g, increasing power or applying binary accept/reject criteria) are either insufficient or misguided. The solution is to embrace the uncertainty and messiness in our data. We need to develop a good intuition of this uncertainty and communicate it faithfully to our peers. I give tips for conveying and interpreting interval estimates in our papers in a honest and truthful way.
        </div>

        <div class="tbio"><b>Bio:</b>
        Pierre Dragicevic is working in the Aviz team at Inria in France as a permanent research scientist. He studies information visualization and human-computer interaction. He co-signed many research articles with p-values until he got frustrated and discovered a vast literature on statistical reform. In 2014 he banished p-values from all his publications. Since then, he has been promoting an approach to better statistical communication based on charts, interval estimation and nuanced interpretations.
        </div>
    </div>
</div>

### Challenge Speaker
<div class="talk">
    <div class="ttitle">A Introduction to microbiome data visualization
        <!--{% if talk.slides %}
        <span class="tspeaker" style="float: right;"><a href="{{ site.baseurl }}/files/{{talk.slides}}">Download
            Slides</a></span>
        {% endif %}-->
    </div>
    <div><span class="tspeaker"><a href="http://franzosa.net/">Eric Franzosa</a></span>, <span> Harvard School of Public Health </span></div>

    <div class="tportrait"><img src="{{ site.baseurl}}/images/speakers/eFranzosa.jpg" style="width: 250px;" alt="E.Franzosa">
    </div>

    <div class="tbioabstract">

        <div class="tabstract"><b>Abstract:</b>
        This talk will introduce the concepts and challenges of visualizing the results of microbial community (microbiome) sequencing projects. We will begin with an overview of current amplicon - and shotgun sequencing-based approaches for generating taxonomic and functional profiles of microbiome samples. We will then discuss methods for effectively representing high-dimensional microbiome profiles in low-dimensional space (ordination). This will include an introduction to the ecological distance measures that underlie microbiome ordination and distinguish it from other dimensionality reduction applications. Following this, we will cover additional topics relevant to visualization of microbiome data, including embedding stratified microbiome profiles on phylogenetic trees, and methods for augmenting common graphical data representations to handle the "quirks" of microbiome data (including compositionality, zero-inflation, and high dynamic range). We will close with an overview of existing platforms for visualizing microbiome data and by pointing out open challenges in the field.
        </div>

        <div class="tbio"><b>Bio:</b>
        Eric received a Bachelors in Biophysics (2006) from Brown University and a Ph.D. Bioinformatics (2011) from Boston University, where he studyied protein structure-evolution relationships with Prof. Brandon Xia. In 2012 he joined Curtis Huttenhower's lab at Harvard School of Public Health/Broad Institute to research metatranscriptomics and microbiome personalization. In 2015 he transitioned to staff scientist role in the Huttenhower lab and is currently researching efficient methods for microbiome functional profiling (HUMAnN2) and expanding the Human Microbiome Project. He is also involved in various Microbiome and bioinformatics workshops and education initiatives. 
        </div>
    </div>
</div>


*(more program details and schedule to be announced shortly)*
